### PR TITLE
Fix jump to date functionality and displaying events recycler view.

### PR DESCRIPTION
**Pull request by P1200**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -25,4 +25,95 @@
     <string name="event_note">Note</string>
     <string name="drawer_open">open</string>
     <string name="drawer_close">close</string>
+    <string name="large_text">
```
Comment (Amaimon208) [inline app/src/main/res/values/strings.xml:28]: I guess that large_text was used for testing, do we still need it?

File: app/src/main/java/com/calendar/YearViewFragment.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container,
+                             @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_year_view, container, false);
+
+        monthsRecyclerView.add(view.findViewById(R.id.calendarRecyclerView1));
```
Comment (Amaimon208) [inline app/src/main/java/com/calendar/YearViewFragment.java:47]: Wouldn't be better to use loop with getIdentifier to clean the code?

File: app/src/main/java/com/calendar/MainActivity.java [hunk 3]
```diff
@@ -146,6 +147,10 @@ public boolean onOptionsItemSelected(@NonNull MenuItem item) {
                         (datePicker1, year, month, day) -> {
                             CalendarUtils.selectedDate = LocalDate.of(year, month + 1, day);
                             onResume();
+                            Fragment calendarFragment = fragmentManager.findFragmentByTag("calendarFragment");
+                            if(calendarFragment != null && calendarFragment.isAdded()) {
```
Comment (Amaimon208) [inline app/src/main/java/com/calendar/MainActivity.java:151]: I would propose to move this condition from if to new method.

File: app/src/main/java/com/calendar/MainActivity.java [hunk 4]
```diff
@@ -104,6 +138,8 @@ public boolean onCreateOptionsMenu(Menu menu) {
     @Override
     public boolean onOptionsItemSelected(@NonNull MenuItem item) {
         switch (item.getItemId()) {
+            case android.R.id.home:
+                return drawerToggle != null && drawerToggle.onOptionsItemSelected(item) || super.onOptionsItemSelected(item);
```
Comment (Amaimon208) [inline app/src/main/java/com/calendar/MainActivity.java:142]: I would propose to move this return statement to new method. To better explain its meaning.

**Review by Amaimon208 — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (P1200) [reply app/src/main/res/values/strings.xml:28]: Removed.

File: app/src/main/java/com/calendar/YearViewFragment.java (hunk 2 above)
Comment (P1200) [reply app/src/main/java/com/calendar/YearViewFragment.java:47]: I don't see any possibility to use loop here, i just moved code to the new method.

File: app/src/main/java/com/calendar/MainActivity.java (hunk 4 above)
Comment (P1200) [reply app/src/main/java/com/calendar/MainActivity.java:142]: Moved to method.

File: app/src/main/java/com/calendar/MainActivity.java (hunk 3 above)
Comment (P1200) [reply app/src/main/java/com/calendar/MainActivity.java]: Moved to a method.

**Pull request merged**